antora-playbook.yml

```diff
@@ -91,7 +91,7 @@ asciidoc:
     prod-operator: che-operator
     prod-operator-image-name: che-operator
     prod-prev-ver-major: 6
-    prod-prev-ver: 7.16
+    prod-prev-ver: 7.17
```

Comment:
current release of Che is 7.19, so prev version is 7.18, right?

Reply:
Seems we have a version maintenance issue ...

* `prod-ver: 7.18` in `antora-playbook.yml` -> should be 7.19
* `7.20.0-SNAPSHOT` in `VERSION` -> should be 7.19 also?

Reply:
7.20 is what's in master branch; 7.19.2 will be out later this week, with 7.20 the week after.

Reply:
The *published* docs is master. We don't have any published docs for a released version.

Reply:
so you want docs for master version only?